### PR TITLE
xerces-c, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Tests fail for x86_gcc2, marked it as unsupported ...

File: dev-libs/xerces_c/patches/xerces_c-3.2.1.patchset [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+diff --git a/src/xercesc/util/XMLDateTime.cpp b/src/xercesc/util/XMLDateTime.cpp
+index 07dac1a..3345481 100644
+--- a/src/xercesc/util/XMLDateTime.cpp
++++ b/src/xercesc/util/XMLDateTime.cpp
+@@ -26,6 +26,10 @@
+ #    include <config.h>
+ #endif
+ 
++#if defined(__HAIKU__)
++#define _BSD_SOURCE
```
Comment (alaviss) [inline dev-libs/xerces_c/patches/xerces_c-3.2.1.patchset]: If you add `-D_BSD_SOURCE` into `CPPFLAGS` instead, this wouldn't be necessary :)

File: dev-libs/xerces_c/xerces_c-3.2.1.recipe [hunk 2]
```diff
@@ -64,31 +71,37 @@ BUILD_PREREQUIRES="
...
 BUILD()
 {
-	autoreconf -fi
-	LDFLAGS="-lnetwork -lbsd" \
-		CFLAGS="-D_BSD_SOURCE" \
-		runConfigure ./configure
-
+	export LDFLAGS="-lnetwork -lbsd"
+	export CPPFLAGS="-D_BSD_SOURCE"
+	autoreconf -vfi
```
Comment (alaviss) [inline dev-libs/xerces_c/xerces_c-3.2.1.recipe]: Is this necessary?

Comment (Begasus) [reply dev-libs/xerces_c/xerces_c-3.2.1.recipe]: Need to re-check ...

**Pull request merged**